index.js

```diff
@@ -215,8 +215,15 @@ exports.stringify = (obj, options) => {
 };
 
 exports.parseUrl = (input, options) => {
+	let hash = '';
+	const qSplits = input.split('?').map(part => {
+		const sSplits = part.split('#');
+		hash = sSplits[1] || hash;
+		return sSplits[0];
+	});
 	return {
-		url: input.split('?')[0] || '',
-		query: parse(extract(input), options)
+		url: qSplits[0],
+		hash,
```

Comment:
The hash should still just be part of the URL and not an exposed property.

Reply:
IMHO, Exposing hash makes comparability with `{pathname, hash, search}` pattern in other lib like [react-router](https://medium.com/@pshrmn/a-simple-react-router-v4-tutorial-7f23ff27adf)

Reply:
The point of the method is to allow you to get the query from an URL. I don't want to overload it with lots of other URL parsing stuff. It would also be a breaking change, as people might depend on getting the hash in the `url` property, and I'm not interested in doing a major release just for this.